### PR TITLE
examples: Update Kubernetes from v1.7.3 to v1.7.5

### PR DESCRIPTION
**Pull request by dghubble**
* Update bootkube version from v0.6.1 to v0.6.2
* Update manual Kubernetes cluster example from v1.7.3 to v1.7.5
* Update Terraform Kubernetes cluster example from v1.7.3 to v1.7.5
   * Switch Terraform example to use Typhoon project's module instead of a copy in this repo
   * Includes support for Calico and Flannel

### CODE REVIEW

File: examples/terraform/bootkube-install/cluster.tf [hunk 1]
```diff
@@ -1,15 +1,18 @@
-// Self-hosted Kubernetes cluster
+// Kuberentes cluster
```
Comment (diegs) [inline examples/terraform/bootkube-install/cluster.tf]: Kubernetes

File: examples/terraform/bootkube-install/cluster.tf [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 module "cluster" {
-  source = "../modules/bootkube"
+  source = "git::https://github.com/poseidon/typhoon//bare-metal/container-linux/kubernetes?ref=a441f5c6e03e46557e3eafa6f244f649bb00095f"
```
Comment (diegs) [inline examples/terraform/bootkube-install/cluster.tf]: What's this?

Comment (diegs) [reply examples/terraform/bootkube-install/cluster.tf]: Cool

File: examples/terraform/bootkube-install/cluster.tf (hunk 1 above)
Comment (dghubble) [reply examples/terraform/bootkube-install/cluster.tf]: fixed

**Pull request merged**